src/mutations/ScheduledEventMutations.ts

```diff
@@ -263,6 +263,18 @@ export default class ScheduledEventMutations {
       action
     );
 
+    const isCompletedAction = action === ProposalBookingFinalizeAction.COMPLETE;
+
+    eventBus.publish({
+      type: isCompletedAction
+        ? Event.PROPOSAL_BOOKING_TIME_COMPLETED
+        : Event.PROPOSAL_BOOKING_TIME_REOPENED,
```

Comment:
Just have a question, because it's not clear when reading the flow of the code.

If I call `finalize` for the scheduled event, in what scenario do we need to dispatch an event PROPOSAL_BOOKING_TIME_REOPENED. It feels a little counterintuitive.

Reply:
Yes probably you are right, this is not the most intuitive logic here. If the `action` is NOT `COMPLETE` we dispatch the `PROPOSAL_BOOKING_TIME_REOPENED`. Maybe I can put this in a small helper function like `getFinalizeEventType` where it will resolve the correct type a bit better.